### ISSUE TITLE
Traffic lights show problems

### ISSUE THREAD

**Issue by HIGHGO77** [CitiesSkylinesMods/TMPE#1191]
### Describe your idea
I wonder how to make the traffic light icon display while the orange circle in the middle is not displayed. Whether it can be implemented through other modules or directly through this mod.

### Notes or questions?
<img width="605" alt="屏幕截图 2021-11-11 172815" src="https://user-images.githubusercontent.com/46513979/141274129-d4f6b4fa-1a2a-4320-883c-81282a53b982.png">

**Comment by kianzarrin** [comment]
it wouldn't be hard to implement but too many options can be confusing. is it really that important?

**Comment by HIGHGO77** [comment]
Yes, this option is important to me right now. I would appreciate it if you could help me implement this feature.

**Comment by krzychu124** [comment]
> it wouldn't be hard to implement but too many options can be confusing. is it really that important?

hmm maybe just add toggle to already existing traffic light overlay (Overlays tab), similar as we have for "turn on red" at Maintenance tab 🤔

**Comment by HIGHGO77** [comment]
I don't know what you mean, the "turn on red" in the maintenance tab doesn't eliminate the big circle in the middle，nor can it be achieved in “Overlays tab”
